### DISCUSSION TITLE
Add GhostRider Support

### DISCUSSION THREAD

**Discussion by Raptor3um** [coinfoundry/miningcore#855] [Ideas]
Please add GhostRider algorithm support (from Raptoreum).
https://github.com/npq7721/gr_hash

Miner support is growing quickly and it is currently the most profitable CPU coin. GPU miner is not too far off as far as I know.

I made this again here in discussions as I received an email that issues were now to be posted in discussions.

**Comment by calvintam236** [comment]
duplicate of #852
